types/react-big-calendar/index.d.ts

```diff
@@ -4,8 +4,9 @@
 //                 Austin Turner <https://github.com/paustint>
 //                 Krzysztof Bezrąk <https://github.com/pikpok>
 //                 Sebastian Silbermann <https://github.com/eps1lon>
+//                 Paul Potsides <https://github.com/strongpauly>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 2.9
```

Comment:
Was this necessary?

Reply:
I don't think so.  I've rolled it back to 2.8